contracts/PetPark.sol

```diff
@@ -4,4 +4,72 @@ pragma solidity ^0.8.0;
 
 contract PetPark {
 
-}
\ No newline at end of file
+    address owner;
+
+    enum AnimalType {None, Fish, Cat, Dog, Rabbit, Parrot}
+
+    struct Person {
+        uint age;
+        uint gender;
+        AnimalType AnimalBorrowed;
+        bool hasBorrowed;
+        }
+
+    mapping (AnimalType => uint) CountByAnimalType;
+    mapping (address => Person) PersonByAddress;
+    mapping (address => uint) BorrowedByAddress;
+
+    event Added (AnimalType indexed animalType, uint indexed count);
+    event Borrowed (uint indexed animalType);
+    event Returned  (uint indexed animalType);
+
+    constructor() {
+        owner = msg.sender;
+    }
+    
+    modifier onlyOwner() {
+        require(msg.sender == owner, "Not an owner.");
+        _;
+        }
+    
+    function add (uint _AnimalType, uint _AnimalCount) public onlyOwner {
+        AnimalType ThisAnimalType = AnimalType(_AnimalType);
+        require(ThisAnimalType != AnimalType.None,"Invalid animal type");
+        CountByAnimalType[ThisAnimalType] = _AnimalCount;
```

Comment:
This should be `CountByAnimalType[ThisAnimalType] += _AnimalCount;` since we adding these animals